src/storage/v2/inmemory/label_property_index.cpp

```diff
@@ -202,6 +202,12 @@ void InMemoryLabelPropertyIndex::Iterable::Iterator::AdvanceUntilValid() {
       continue;
     }
 
+    if (self_->transaction_->original_start_timestamp.has_value()) {
+      if (index_iterator_->timestamp >= self_->transaction_->original_start_timestamp.value()) {
+        continue;
+      }
+    }
+
```

Comment:
This logic gets repeated a lot, can we define it in a single place?

Reply:
added it under `indices_utils.hpp`